feat(router-store): add routerState config option (#1847)
ngrx/platform: projects/ngrx.io/content/guide/router-store/configuration.md

  stateKey?: string | Selector&lt;any, RouterReducerState&lt;T&gt;&gt;;
  serializer?: new (...args: any[]) => RouterStateSerializer;
  navigationActionTiming?: NavigationActionTiming;

>   state?: RouterState;

Reviewer: ```suggestion
  routerState?: RouterState;
```

  navigationActionTiming: NavigationActionTiming.PostActivation,
});
</code-example>

> ## routerState

Reviewer: ```suggestion
## Router State Snapshot
```

This property decides which router serializer should be used. If there is a custom serializer provided, it will use the provided serializer. `routerState` also sets the metadata on dispatched `@ngrx/router-store` action.

### RouterState.Full

When this property is set to `RouterState.Full`, `@ngrx/router-store` will use the `DefaultRouterStateSerializer` serializer to serialize the Angular router event.

> The metadata on the action will contain the Angular router event, e.g. NavigationStart` and `RoutesRecognized`.

Reviewer: ```suggestion
The metadata on the action will contain the Angular router event, e.g. `NavigationStart` and `RoutesRecognized`.
```

<code-example header="app.module.ts">
StoreRouterConnectingModule.forRoot({
  routerState: RouterState.Full,
});
</code-example>

### RouterState.Minimal

> `RouterState.Minimal` will use the `MinimalRouterStateSerializer` serializer to serialize Angular router event.

Reviewer: ```suggestion
`RouterState.Minimal` will use the `MinimalRouterStateSerializer` serializer to serialize the Angular Router's `RouterState` and `RouterEvent`.
```

The metadata on the action consists of the navigation id and the url.

> ```ts

Reviewer: ```suggestion
```json
```
